team-manager/src/main/java/ua/knu/knudev/teammanager/domain/AccountProfile.java

```diff
@@ -24,4 +28,33 @@ public class AccountProfile {
 
     @Column(nullable = false)
     private String middleName;
+
+    //    todo add validation by @knu.ua
+    @Column(nullable = false)
+    private String email;
+
+    @Column
+    private String avatar;
+
+    @Column(nullable = false)
+    private String password;
+
+    @Column(nullable = false)
+    @Enumerated(EnumType.STRING)
+    private AccountRole accountRole;
+
+    @Column(nullable = false, updatable = false)
+    @CreatedDate
+    private LocalDateTime registrationDate;
+
+    @Column
+    private LocalDateTime lastRoleUpdateDate;
+
+    @OneToOne(cascade = CascadeType.PERSIST)
+    @JoinColumn(name = "department_id", referencedColumnName = "id")
+    private Department department;
+
+    @OneToOne(cascade = CascadeType.PERSIST)
+    @JoinColumn(name = "specialty_id", referencedColumnName = "codeName")
```

Comment:
@JoinColumn(name = "specialty_id", referencedColumnName = "code_name")

code_name, not codeName

Reply:
done